tensorboard/plugins/projector/vz_projector/standalone.html

```diff
@@ -51,6 +48,9 @@
   </style>
 </head>
 <body>
+  <script>
+    <!-- HaCK: GA requires at least one script tag on document.  -->
```

Comment:
nit: is this capitalization intended?

Reply:
Nope but it looks cool (and hackerish!). Fixed it.